seonshine_frontend/src/components/organims/sideBar/constants.ts

```diff
@@ -0,0 +1,38 @@
+import { EventNoteOutlined, ListAltOutlined, PeopleOutline, StorefrontOutlined } from '@mui/icons-material';
+
+import { RoleEnum } from '@/types/user';
+
+export const menuItems = [
```

Comment:
the menu items can be used in multiple places (header...). We need to move to global constants . `constants/menu.ts`